src/js/transforms/DropZone.js

```diff
@@ -7,9 +7,27 @@ var dl = require('datalib'),
     Transform = vg.Transform,
     sg = require('../model/signals');
 
+
+DropZone.prototype = Object.create(Transform.prototype);
+DropZone.prototype.constructor = DropZone;
```

Comment:
This only works because of hoisting -- that'll probably be caught by the linter so it doesn't need to be addressed here, but as a note for @arvind we should avoid extending constructors before they're defined ("before" meaning as we read linearly through a file).